<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TeachBox</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="fun_ListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>60</width>
      <height>531</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>运行</string>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>16</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>状态</string>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item>
     <property name="text">
      <string>设置</string>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
   </widget>
   <widget class="QStackedWidget" name="fun_StackedWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>920</width>
      <height>530</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_3"/>
    <widget class="QWidget" name="page_2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>920</width>
        <height>530</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="nelwork_GroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="title">
       <string>网络设置</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QLabel" name="IP_Label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>60</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>IP</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="IP_LineEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>191</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="port_Label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>60</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>端口</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="port_LineEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>191</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="setNet_Btn">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>150</y>
         <width>91</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="netStatus_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="tmp">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>100</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tmp2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>100</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QComboBox" name="plan_ComboBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>五次多项式</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>T型曲线</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="plan_Label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>规划器</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ctr_Label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>控制律</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="ctr_ComboBox">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>反步控制</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>重力补偿PD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>计算力矩</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="jog_GroupBox">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>259</y>
      <width>180</width>
      <height>330</height>
     </rect>
    </property>
    <property name="title">
     <string>点动</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="J1_Btn1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="J1_Btn2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="J2_Btn1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="J3_Btn1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="J4_Btn1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="J5_Btn1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="J6_Btn1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="J7_Btn1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="J2_Btn2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="J3_Btn2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="J4_Btn2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="J5_Btn2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="J6_Btn2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>240</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="J7_Btn2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>280</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="J1_Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>J1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="J2_Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>J2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="J3_Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>J3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="J4_Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>J4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="J5_Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>J5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="J6_Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>J6</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="J7_Label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>20</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>J7</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="vel_GroupBox">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>10</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>速度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="vel_Btn1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>40</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="vel_Btn2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>40</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="vel_LineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="enable_Btn">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>110</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>enable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disable_Btn">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>110</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>disable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backZero_Btn">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>160</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>回零</string>
    </property>
   </widget>
   <widget class="QPushButton" name="oneStep_Btn">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>160</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>单步</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_Btn">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>210</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_Btn">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>210</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>fun_ListWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>fun_StackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
